l10n_it_account_balance_report/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,7 @@
+**Italiano**
+
+Report PDF e XLS per stato patrimoniale e conto economico a sezioni contrapposte.
```

Comment:
Scusami, mi era sfuggito. 🙏 
Anche questo andrebbe uniformato.

P.s: per quanto riguarda il problema dell'xls non tradotto se riesco provo a darci un'occhiata.

Reply:
P.S.2: dimenticavo, il file it.po non l'ho revisionato, meglio farlo direttamente su weblate dopo il merge del modulo.